GSA/digital-gov-drupal
# Digital-33: Adding Views

## Jira ticket




[DIGITAL-33](https://cm-jira.usa.gov/browse/DIGITAL-33)

## Purpose



This PR is setting up the Drupal Views for rendering lists and setting up the twig templates for each one. 

- includes the twig files, config set up for views and a preprocess file for setting up image path for icons

Please not there are several `todo:` in these files for us to revisit and update once other work is compelted

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup



1. `lando rebuild -y` and clear cache.
2. be sure to log in as an admin as well. 

### QA/Testing instructions

1. in the `page.html.twig` add `{{ drupal_view('<view_name>', 'block_1') }}` and add each of the views created 
2. review view set up to ensure the admin config is set up correctly 
3. also review twig files to ensure set up is correct for migration

## Checklist for the Developer




## Checklist for the Peer Reviewers
